TwitchDownloaderWPF/PageChatRender.xaml.cs

```diff
@@ -511,6 +522,13 @@ private async void SplitButton_Click(object sender, RoutedEventArgs e)
                     return;
                 }
 
+                // Force "enqueue render" if multiple files are selected
+                if (fileNames.Length > 1)
```

Comment:
The partial render button is still clickable despite not being honoured, not super great user experience.

Reply:
Agreed, there's probably a better way to handle that but I'm not sure how.

Reply:
I will just disable the partial render button from being clickable and switch the text of the render button to make it more clear how the program will handle it.
We will never be able to do partial rendering on multiple files at once unless we rewrite the range selection logic.